e2e/cypress/integration/channel_sidebar/more_unreads_position_with_scroll_spec.js

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channel_sidebar
+
+import {getRandomId} from '../../utils';
+
+describe('Channel sidebar', () => {
+    let mainUser;
+    let otherUser;
+    let myTeam;
+
+    // This will ensure a list of channel names that will be alphabetically sorted
+    const channelNames = new Array(20).fill(1).map((value, index) => `scroll${index}`);
+
+    before(() => {
+        cy.apiInitSetup().then(({team, user: firstUser}) => {
+            mainUser = firstUser;
+            myTeam = team;
+
+            cy.apiCreateUser().then(({user: secondUser}) => {
+                otherUser = secondUser;
+                return secondUser.id;
+            }).then((userId) => cy.apiAddUserToTeam(team.id, userId));
+
+            cy.wrap(channelNames).each((name) => {
+                const displayName = `channel-${name}`;
+                cy.apiCreateChannel(team.id, name, displayName, 'O', '', '', false).then((response) => {
```

Comment:
I need to be able to find my channels by name after they are created so I had to disable the 'unique' argument so the randomly generated suffix would not be added to the name each time (https://github.com/mattermost/mattermost-webapp/blob/1bfd268ca1e3802df8dad08f00b442da491d512e/e2e/cypress/support/api_commands.js#L38)